### PR TITLE
MP-397 api resources setup

### PR DESCRIPTION
**Pull request by nessy**
- [x] Step one find a minimal of 100 spelling mistakes ;)

### CODE REVIEW

File: content/api/resources/Invitations.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  "data": [
+      {
+        "type": "invitations",
+        "id": "c9ce29a4-6325-11e7-907b-a6006ad3dba0",
+        "attributes": {
+          "code": "15317471056093",
+          "email": "john@doe.com",
+          "name": "James Johnson",
+          "message": "Hi James, this is your invitation to join myparcel.com",
+          "status": "pending"
```
[the remark this answers is not in the document]
Comment (nessy) [reply content/api/resources/Invitations.md]: ok

File: content/api/resources/carriers.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+#### Sample response
+```http
+HTTP/1.1 200 OK
+Content-Type: application/vnd.api+json
+
+{
+  "data": [
+    {
+      "type": "carriers",
+      "id": "[carrier-id]",
```
Comment (SAMDevelopment) [inline content/api/resources/carriers.md]: When providing sample responses, use actual sample data. So use a uuid and think of a name. Maybe keep it inline with the example values from the API spec.

File: content/api/resources/carriers.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        "name": "[carrier-name]"
+      }
+    }
+  ],
+  "meta": {
+    "total_pages": 1,
+    "total_records": 1
+  },
+  "links": {
+    "self": "/carriers?page[number]=3&page[size]=30",
```
Comment (SAMDevelopment) [inline content/api/resources/carriers.md]: This link cannot exist with 1 page.

File: content/api/resources/carrier-contracts.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+## Carrier Contract
+
+{{< icon fa-file-text-o >}}[API specification](https://docs.myparcel.com/api-specification#/CarrierContracts)
+
+Attribute         | Description
+----------------- | -----------
+currency          | The currency used for all prices for the attached service contracts.
+
+Relation          | Description
```
Comment (SAMDevelopment) [inline content/api/resources/carrier-contracts.md]: "Relation" => "Relationship"

File: content/api/resources/carrier-contracts.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+----------------- | -----------
+carrier           | Carrier offering the contract.
+service_contracts | Contracts for services offered by the carrier, which define prices and options.
+
+## Retrieve carrier contracts
+To get all the contracts of a carrier call the [GET /carrier-contracts/{carrier_contract_id}](https://docs.myparcel.com/api-specification#/CarrierContracts/get_carrier_contracts__carrier_contract_id_) endpoint.
+
+#### Sample request
+```http
+GET /carrier-contracts/{carrier_contract_id} HTTP/1.1
```
Comment (SAMDevelopment) [inline content/api/resources/carrier-contracts.md]: Also use sample data with a sample request.

File: content/api/resources/carrier-contracts.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+```
+
+#### Sample response
+```http
+HTTP/1.1 200 OK
+Content-Type: application/vnd.api+json
+
+{
+  "data": {
+      "type": "carrier-contracts",
```
Comment (SAMDevelopment) [inline content/api/resources/carrier-contracts.md]: Incorrect indentation

File: content/api/resources/carrier-contracts.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+        "currency": "EUR"
+      },
+      "relationships": {
+        "carrier": {
+          "data": {
+            "type": "carriers",
+            "id": "[carrier-id]"
+          },
+          "links": {
+            "related": "http://localhost:8080/v1/carriers/[carrier-id]"
```
Comment (SAMDevelopment) [inline content/api/resources/carrier-contracts.md]: Use either our live url or sandbox as host.

File: content/api/resources/carrier-pudo-locations.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+Carrier PUDO locations are locations where:
+
+- your shipments can be picked up once delivered (Pick Up)
+- you can drop the shipments you want to send (Drop Off)
+
+## Pickup Dropoff Location
+
+{{< icon fa-file-text-o >}}[API specification](https://docs.myparcel.com/api-specification#/Carriers/get_carriers__carrier_id__pickup_dropoff_locations__country_code___postal_code_)
+
+Relation | Description
```
Comment (SAMDevelopment) [inline content/api/resources/carrier-pudo-locations.md]: Why no attributes?

File: content/api/resources/files.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
++++
+title = "Files"
+description = "PDF labels and other file resources returned by the carriers."
+weight = 5
++++
+
+{{< icon fa-file-text-o >}}[API specification](https://docs.myparcel.com/api-specification#/Files)
+
+A file resource can represent all kinds of files like `labels`, `barcodes`, or `logos` that can be available in different file formats.
```
Comment (SAMDevelopment) [inline content/api/resources/files.md]: Don't know if we should mention logos. We might never support that.

File: content/api/resources/files.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+
+## Types
+If you request a file using the default file type `application/vnd.api+json` as Accept header, you will get a list of the supported mime types for that file. This list can contain the following mime types:
+
+- application/vnd.api+json
+- application/pdf
+- image/png
+- image/jpeg
```
Comment (SAMDevelopment) [inline content/api/resources/files.md]: `jpeg` is no longer supported.

File: content/api/resources/files.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+          "mime_type": "application/pdf"
+        }
+      ]
+    },
+    "links": {
+      "self": "/files/[file-id]"
+    }
+  }
+}
+```
```
Comment (SAMDevelopment) [inline content/api/resources/files.md]: Maybe add a sample request do download a file.

File: content/api/resources/Invitations.md [hunk 12]
```diff
@@ -0,0 +1,64 @@
++++
+title = "Invitations"
+description = "Invitations you have sent to friends and the users you have connected this way."
+weight = 6
++++
+
+{{< icon fa-file-text-o >}}[API specification](https://docs.myparcel.com/api-specification#/Invitations)
+
+The invitations are used to invite new users to the MyParcel.com platform. When you create an invitation it will automatically send an email to the provided email address. If the link in this email is used to register a new user, that user will be linked to your account.
```
Comment (SAMDevelopment) [inline content/api/resources/Invitations.md]: "... that user will be linked to your account."
To my account? The developer that made the application that does the request?

**Comment by M4tini** [comment]
I added a new task `MP-569` to describe all endpoints in more detail with (references to) request and response examples.

File: content/api/resources/files.md [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+- image/png
+
+## Retrieve a file in a specific format
+
+{{< icon fa-file-text-o >}}[GET /files/{file_id}](https://docs.myparcel.com/api-specification#/Files/get_files__file_id_)
+
+Use these mime types as Accept header to retrieve the actual file as binary data.
+
+```http
+GET /files/[file-id] HTTP/1.1
```
Comment (SAMDevelopment) [inline content/api/resources/files.md]: Use a uuid instead of "[file-id]"

File: content/api/resources/regions.md [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+
+## Retrieve regions
+
+For an example request and response, check our API specification:<br>
+{{< icon fa-file-text-o >}}[GET /regions](https://docs.myparcel.com/api-specification#/Regions/get_regions)
+
+### Parameters
+
+You can filter the regions on a parent, country_code, region_code and name.
+If you add these filters the call would look something like this:
```
Comment (SAMDevelopment) [inline content/api/resources/regions.md]: "call" => "request"

File: content/api/resources/service-contracts.md [hunk 15]
```diff
@@ -0,0 +1,61 @@
++++
+title = "Service Contracts"
+description = "A service can have one or multiple contracts, this resource specifies the prices and available options."
+weight = 10
++++
+
+A service contract is a combination of a [service](/api/resources/services/) with a [carrier_contract](/api/resources/carrier-contracts/). Each combination contains an unique recipe to send a shipment. One of the weight groups should be applicable, an insurance might be available or already included. Also extra options might be available and increase the base price with a certain amount.
```
Comment (SAMDevelopment) [inline content/api/resources/service-contracts.md]: "Each combination contains an unique recipe to ..." => "Each combination contains **a** unique recipe to ..."

Comment (SAMDevelopment) [reply content/api/resources/service-contracts.md]: "One of the weight groups should be applicable, ..."
Weird sentence.

Comment (SAMDevelopment) [reply content/api/resources/service-contracts.md]: "Also ..." => "Also, ..."

File: content/api/resources/service-options.md [hunk 16]
```diff
@@ -0,0 +1,17 @@
...
+Service options can be added to a [shipment](/api/resources/shipments). The added price depends on the selected [service contract](/api/resources/service-contracts/). Options are a generic resource since many carriers offer the same options, like "delivery on saturday".
+
+## Service Option
+
+{{< icon fa-file-text-o >}}[API specification](https://docs.myparcel.com/api-specification#/ServiceOptions)
+
+Attribute     | Description
+------------- | -----------
+name          | Option name, useful for displaying to users.
+category      | If the option belongs to a specific category like `delivery-day` or `handover-method`, useful to filter on and display options in groups.
```
Comment (SAMDevelopment) [inline content/api/resources/service-options.md]: "If the option belongs to a specific ..."
Weird sentence.

File: content/api/resources/service-options.md [hunk 17]
```diff
@@ -0,0 +1,17 @@
...
+code          | Option code, useful for programmers.
```
Comment (SAMDevelopment) [inline content/api/resources/service-options.md]: lol

File: content/api/resources/shipments.md [hunk 18]
```diff
@@ -0,0 +1,77 @@
...
+## Shipment
+
+{{< icon fa-file-text-o >}}[API specification](https://docs.myparcel.com/api-specification#/Shipments)
+
+Attribute                    | Description
+---------------------------- | -----------
+recipient_address            | [Address](/api/resources/addresses) object used as delivery address if no pickup location is specified.
+sender_address               | [Address](/api/resources/addresses) object used as return address.
+pickup_location              | [PUDO location](/api/resources/carrier-pudo-locations) object containing the `code` and `address` of the location.
+description                  | Short note, printed on the shipping label if specified in the shop settings (and if allowed by the carrier).
```
Comment (SAMDevelopment) [inline content/api/resources/shipments.md]: This is not implemented yet.

**Pull request merged**